Userland/Libraries/LibWeb/DOM/ElementFactory.cpp

```diff
@@ -474,6 +476,44 @@ static WebIDL::ExceptionOr<JS::GCPtr<SVG::SVGElement>> create_svg_element(JS::Re
     return nullptr;
 }
 
+static WebIDL::ExceptionOr<JS::GCPtr<MathML::MathMLElement>> create_mathml_element(JS::Realm& realm, Document& document, QualifiedName qualified_name)
+{
+    auto const& local_name = FlyString::from_deprecated_fly_string(qualified_name.local_name()).release_value_but_fixme_should_propagate_errors();
+
+    if (local_name == MathML::TagNames::annotation
```

Comment:
Nit-pick: `local_name.is_one_of(...)` might be a little less verbose, but it's no big deal, so feel free to ignore this.